python/paddle/v2/fluid/tests/book_memory_optimization/test_memopt_fit_a_line.py

```diff
@@ -24,15 +24,21 @@
 fluid.default_startup_program().random_seed = 111
 
 x = fluid.layers.data(name='x', shape=[13], dtype='float32')
-
-y_predict = fluid.layers.fc(input=x, size=1, act=None)
-
 y = fluid.layers.data(name='y', shape=[1], dtype='float32')
 
-cost = fluid.layers.square_error_cost(input=y_predict, label=y)
-avg_cost = fluid.layers.mean(x=cost)
+places = fluid.layers.get_places(device_count=2, device_type='CPU')
+pd = fluid.layers.ParallelDo(places)
```

Comment:
Could you also test it along with nccl? Basically, change `fluid.layers.ParallelDo(places)` to `fluid.layers.ParallelDo(places, use_nccl=True)`

https://github.com/PaddlePaddle/Paddle/blob/7a9098a60e6f65bc714640f1cc94b89f89cfd8fe/python/paddle/v2/fluid/tests/unittests/test_parallel_op.py#L112

Reply:
Sure, I will make test with NCCL.